### DISCUSSION TITLE
Laragon.ini configurations

### DISCUSSION THREAD

**Discussion by kawishbit** [leokhoa/laragon#556] [Q&A]
Hi everyone, 
I was trying to find the list of configurations that are available in the laragon.ini file; however, I can't find any documentation for this.
I haven't been using Laragon for a while, and I just noticed that the old forum has been taken down.

Does anyone know where I can find the list of configurations for the laragon.ini file?

Thanks for your help.

**Comment by Pen-y-Fan** [comment]
The laragon.ini file is more for internal use by Laragon, for example, it will automatically update the file when Apache is updated.

In the years I have used Laragon, the only item I've changed in that file is the editor, as I wanted to use my downloaded version of Notepad++, which has plugins.

```ini
Editor="C:\Program Files\Notepad++\Notepad++.exe"
```

Other, than that I never manually edit that file.

What configurations are you interested in? What are you trying to do?

**Reply by kawishbit** [reply]
Thanks for the quick reply, 
I'm interested in changing the terminal for laragon, from cmder to windows terminal, my first intuition was to find out whether it can be changed from the laragon.ini config or not

**Comment by MichalJarocinski** [comment]
How to make such a change (e.g. Notepad++ to Notepad4) in laragon 8? If inside laragon.ini then in which section? Is the path containing spaces to be enclosed in quotation marks?
